BuildAssets/test-linux-packages.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash
 
-# This script requires environment variables to run:
-#   OCTOPUS_CLI_SERVER, OCTOPUS_CLI_API_KEY, OCTOPUS_SPACE, OCTOPUS_EXPECT_ENV
-# You must also supply either:
-#   TEST_QUICK (to skip some distributions)
-# or
-#   REDHAT_SUBSCRIPTION_USERNAME, REDHAT_SUBSCRIPTION_PASSWORD (to support RHEL tests)
+if [[ -z "$OCTOPUS_CLI_SERVER" || -z "$OCTOPUS_CLI_API_KEY" || -z "$OCTOPUS_SPACE" || -z "$OCTOPUS_EXPECT_ENV" ]]; then
```

Comment:
as above re one line per var

Reply:
Okay done, but with a joint explanation for the Octopus variables as they all relate.